internal/mode/static/nginx/config/servers.go

```diff
@@ -487,6 +493,50 @@ func generateProxySetHeaders(filters *dataplane.HTTPFilters) []http.Header {
 	return append(proxySetHeaders, headers...)
 }
 
+func generateAddResponseHeaders(filters *dataplane.HTTPFilters) []http.Header {
+	headers := make([]http.Header, len(baseHeaders))
```

Comment:
This function also handles `baseHeaders`.

Reply:
I don't think we need to handle `baseHeaders` in this function. The `baseHeaders` are request headers and are handled by `generateProxySetHeaders`.

Reply:
Update https://github.com/nginxinc/nginx-gateway-fabric/pull/1494/commits/bcfa113486f0f38caa938a3051a8064fed3908a7